plugins/index.ts

```diff
@@ -95,9 +111,14 @@ const deployOrUpgradeProxy = async <
     contractCode === '0x' ||
     process.env.FORCE_PROXY_DEPLOYMENT ||
     typeof maybeProxyAddress !== 'string';
-  if (shouldDeployProxy) {
+  if (
+    shouldDeployProxy &&
+    !['bridgedpolygonnori', 'nori', 'lockednori'].includes(
+      contractName.toLowerCase()
+    )
+  ) {
     hre.trace('Deploying proxy and instance', contractName);
-    const fireblocksSigner = signer as FireblocksSigner;
+    const fireblocksSigner = signer as any as FireblocksSigner;
```

Comment:
Possible to fix this before merging?

Reply:
yep - it can removed now that our impersonation code isn't active